roles/swift-account/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
             dest=/etc/swift/account-server.conf owner=swift group=swift
   notify: restart swift-account service
 
+- name: trigger restart on upgrades
```

Comment:
@j2sol Handler I referred to gets used during Swift source install.

I completely agree with you that restarting all services may not make sense (especially if services are hosted on different node). However, I just wanted to raise point that not all "
affected" services would be restarted.

I can create a separate PR to fix that.